src/metabase/driver/common.clj

```diff
@@ -122,7 +122,7 @@
     :default      false}
    {:name         "tunnel-host"
     :display-name (deferred-tru "SSH tunnel host")
-    :helper-text  (deferred-tru "The hostname that you use to connect to connect to SSH tunnels.")
```

Comment:
This was already fixed (https://github.com/metabase/metabase/pull/19833) and merged into master.

Reply:
@nemanjaglumac Thanks, branch rebased o/